app/controllers/profiles_controller.rb

```diff
@@ -45,7 +45,7 @@ def dashboard
     @passengers = Passenger.where(journey_id: Journey.where(user: @user))
 
 
-    else
+    elsif @user.cars[0]
```

Comment:
I added `exists`, it seems more efficient: https://semaphoreci.com/blog/2017/03/14/faster-rails-how-to-check-if-a-record-exists.html